infer/SigSuggestion.cc

```diff
@@ -89,7 +89,9 @@ void extractSendArgumentKnowledge(core::Context ctx, core::LocOffsets bindLoc, c
         snd->receiverLoc,
         snd->argLocs,
     };
-    core::DispatchArgs dispatchArgs{snd->fun, locs, args, snd->recv.type, snd->recv.type, snd->recv.type, snd->link};
+    auto originForUninitialized = core::Loc::none();
+    core::DispatchArgs dispatchArgs{snd->fun,       locs,           args,      snd->recv.type,
+                                    snd->recv.type, snd->recv.type, snd->link, originForUninitialized};
```

Comment:
It should be possible to grab the owner loc. The one call site to this method is in a function that has `methodSymbol` (for the owning method) in scope.

Reply:
Though I guess technically it doesn't matter because this _should_ only affect error messages? And we're throwing away any errors that happen as a result of this fake dispatch. So alternatively, I'd be fine if you put a quick comment saying why it's ok to use use Loc::none here.